minifi-docs/src/main/markdown/System_Admin_Guide.md

```diff
@@ -174,10 +174,11 @@ for example "10 sec" or "10 MB", not simply "10".
 
 The first section of config.yml is for naming and commenting on the file.
 
- Property | Description
- -------- | ---
-name      | The name of the file.
-comment   | A comment describing the usage of this config file.
+ Property             | Description
+--------------------- | -----------
+MiNiFi Config Version | The version of the configuration file.
```

Comment:
This should include a little more information for the user to understand what they should put and why. Like that there is currently only 1 version and the default value will be version 1.